src/x3d-loader.js

```diff
@@ -30,8 +30,7 @@ module.exports = function (THREE) {
 
 			var scope = this;
 
-			var loader = new THREE.XHRLoader(this.manager);
```

Comment:
XHRLoader has been deprecated in favor of FileLoader.

`setCrossOrigin` has also been removed (https://github.com/mrdoob/three.js/commit/a88a3e62d73b5964f57899eb60d5fd9291bae1f4#diff-5705be950df59ac0364d8ffe954313f6)